### PR TITLE
Enable spell test

### PR DESCRIPTION
**Pull request by m-kuhn**
## Description

Enables the lost spelling test

### CODE REVIEW

File: .github/workflows/code_layout.yml [hunk 1]
```diff
@@ -120,7 +120,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Indentation Test
+      - name: Install Requirements
+        run: |
+          sudo apt install -y \
+              unbuffer \
```
Comment (m-kuhn) [inline .github/workflows/code_layout.yml:126]:
```suggestion
              expect \
```

**Comment by 3nids** [comment]
It looks like the test will be run over all the files which will take ages.
It would be better to perform the test on modified files only

**Comment by m-kuhn** [comment]
@3nids done

- what would you propose to fix the current error?
- what's required to make it show the correction suggestion?

**Comment by 3nids** [comment]
your last try sounds good, I think you have to merge to see it running.

**Comment by m-kuhn** [comment]
It works, it already prints out an error inside the changed file 

```
Running spell check on files: .github/workflows/code_layout.yml 
170:        run: ./tests/code_layout/test_sipfiles_uptodate.sh
some errors have been found.
##[error]Process completed with exit code 1.
```

 - This needs to be fixed before merging (`# spellok`?)
 - Do you know if it is possible to get also the correction suggestions from the spell checker

**Comment by 3nids** [comment]
#spellok should have been good, I renamed the script instead.

**Pull request merged**